### PR TITLE
fix: ai instance deletion

### PR DESCRIPTION
**Pull request by gsppe**
Fixes instance deletion issues. Depends on https://github.com/webstudio-is/webstudio/pull/2467

### CODE REVIEW

File: apps/builder/app/builder/features/ai/ai-fetch-result.ts [hunk 1]
```diff
@@ -85,6 +65,31 @@ export const fetchResult = async (
...
+    components: availableComponentsNames,
+    projectId: project?.id,
+    instanceId: selectedInstanceSelector?.[0],
+    prompt,
+  };
+
+  // @todo Future commands might not require all the requestParams above.
+  // When that will be the case, we should revisit the validatin below.
+  if (requestParams.instanceId === undefined) {
+    return ["You must select an element."];
```
Comment (kof) [inline apps/builder/app/builder/features/ai/ai-fetch-result.ts:84]:
```suggestion
    return ["Please select an instance on the canvas."];
```

**Review by kof — APPROVED** [review]

File: apps/builder/app/builder/features/ai/ai-fetch-result.ts [hunk 2]
```diff
@@ -85,6 +65,31 @@ export const fetchResult = async (
...
+  }
+  if (
+    RequestParamsSchema.omit({ command: true }).safeParse(requestParams)
+      .success === false
+  ) {
+    return ["Invalid prompt data"];
```
Comment (kof) [inline apps/builder/app/builder/features/ai/ai-fetch-result.ts:90]: not sure when this is the case, but if its possible to get this as a user-facing issue, we might want a better message

Comment (giuseppeg) [reply apps/builder/app/builder/features/ai/ai-fetch-result.ts:90]: do you have any message suggestion for when (for whatever reason) the requestParams is not valid?

Comment (kof) [reply apps/builder/app/builder/features/ai/ai-fetch-result.ts:90]: well if we don't know anything that we can tell the user then I can't improve it

**Review by TrySound — APPROVED** [review]

**Pull request merged**